### PR TITLE
Added support for displaying the elapsed time (etime) of a process

### PR DESCRIPTION
**Pull request by dastergon**
The PR introduces the display of the elapsed time of a process, indicating how long the process has been running for. 

According to the [ps man page](http://linuxcommand.org/lc3_man_pages/ps1.html):
```
etime  ELAPSED  elapsed time since the process was started, in the form [[dd-]hh:]mm:ss.
```

Example:

```
$ gops 849
parent PID:     1
threads:        16
memory usage:   0.066%
cpu usage:      0.114%
username:       dastergon
cmd+args:       /usr/local/opt/etcd/bin/etcd
elapsed time:   02-23:48:13
local/remote:   127.0.0.1:2380 <-> :0 (LISTEN)
local/remote:   127.0.0.1:2379 <-> :0 (LISTEN)
local/remote:   127.0.0.1:49189 <-> 127.0.0.1:2379 (ESTABLISHED)
local/remote:   127.0.0.1:2379 <-> 127.0.0.1:49189 (ESTABLISHED)
```
*Note: I have already [sent a PR](https://github.com/shirou/gopsutil/pull/516) to gopsutil for addition of this functionality, but I am not certain whether it will land on master or not. Consequently, I decided to use the code straight from the PR for this implementation. If the gopsutil PR  gets merged in the future, I will send a follow-up PR to use gopsutil's function instead.*

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -156,6 +161,17 @@ func processInfo(pid int) {
 	}
 }
 
+// ElapsedTime shows the elapsed time of the process indicating how long the
+// process has been running for.
+func ElapsedTime(p *process.Process) (string, error) {
```
Comment (rakyll) [inline main.go]: You don't have to export this function.

elapsedTime

File: main.go [hunk 2]
```diff
@@ -231,3 +247,22 @@ func max(i, j int) int {
 	}
 	return j
 }
+
+// fmtEtimeDuration formats etime's duration based on ps' format:
```
Comment (rakyll) [inline main.go]: Can you add a link where the format is specified?

File: README.md [hunk 3]
```diff
@@ -90,6 +90,7 @@ memory usage:	0.199%
 cpu usage:	0.139%
 username:	jbd
 cmd+args:	/Applications/Splice.app/Contents/Resources/Splice Helper.app/Contents/MacOS/Splice Helper -pid 5985
+elapsed time:   02-23:48:13
```
Comment (rakyll) [inline README.md]: There is an extra space after `elapsed time:`.

**Comment by dastergon** [comment]
Hi @rakyll,

Thanks for the feedback. I fixed the proposed changes, rebased, and squashed the commit history to avoid having multiple commits.

File: main.go (hunk 2 above)
Comment (dastergon) [reply main.go]: added it below but Github still shows this suggestion as active.

**Comment by dastergon** [comment]
Hey @rakyll,

I rebased the PR to catch the latest history. If you believe I addressed all the suggestions, we could merge it.

**Comment by rakyll** [comment]
Can you rebase please?

**Comment by dastergon** [comment]
@rakyll rebased the branch.

**Comment by rakyll** [comment]
Thanks much!

**Pull request merged**